tests/user_examples/extra_src/shared/general_continuum/continuum_dynamics/continuum_dynamics_complex.hpp

```diff
@@ -70,5 +70,125 @@ void BaseShearStressRelaxation2ndHalfWithWall<BaseShearStressRelaxation2ndHalfTy
     }
     this->velocity_gradient_[index_i] += velocity_gradient;
 }
+
+//============================================Plasticity===========================================//
+//=================================================================================================//
+//============BaseStressRelaxation1stHalfWithWall & BaseStressRelaxation2ndHalfWithWall============//
+//=================================================================================================//
+template <class BaseStressRelaxation1stHalfType>
+Vecd BaseStressRelaxation1stHalfWithWall<BaseStressRelaxation1stHalfType>::computeNonConservativeAcceleration(size_t index_i)
+{
+    return this->acc_prior_[index_i];
+}
+
+template <class BaseStressRelaxation1stHalfType>
+void BaseStressRelaxation1stHalfWithWall<BaseStressRelaxation1stHalfType>::interaction(size_t index_i, Real dt)
+{
+    BaseStressRelaxation1stHalfType::interaction(index_i, dt);
+
+    Vecd acc_prior_i = computeNonConservativeAcceleration(index_i);
+    Vecd acceleration = acc_prior_i;
```

Comment:
why not directly :  Vecd acceleration = computeNonConservativeAcceleration(index_i);

Reply:
Cannot directly use `Vecd acceleration = computeNonConservativeAcceleration(index_i);` here, since we still need to use `acc_prior_i` later in this class.
https://github.com/Shuaihao-Zhang/SPHinXsys/blob/cfdc472f24af828cc15ba96b4ecbebc02eee65ff/tests/user_examples/extra_src/shared/general_continuum/continuum_dynamics/continuum_dynamics_complex.hpp#L107